dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/curator/CuratorZookeeperClient.java

```diff
@@ -296,6 +299,15 @@ public void nodeChanged() throws Exception {
 
     static class CuratorWatcherImpl implements CuratorWatcher {
 
+        private static final ExecutorService CURATOR_WATCHER_EXECUTOR_SERVICE = Executors.newSingleThreadExecutor(new ThreadFactory() {
+            @Override
+            public Thread newThread(Runnable r) {
+                Thread thread = new Thread(r);
+                thread.setName("Dubbo-CuratorWatcher-Thread");
+                return thread;
+            }
+        });
+

```

Comment:
Replace to use `org.apache.dubbo.common.threadpool.manager.FrameworkExecutorRepository#getSharedExecutor`.

```java
getUrl().getOrDefaultFrameworkModel().getBeanFactory()
            .getBean(FrameworkExecutorRepository.class).getSharedExecutor();
```

Reply:
> Replace to use `org.apache.dubbo.common.threadpool.manager.FrameworkExecutorRepository#getSharedExecutor`.
> 
> ```java
> getUrl().getOrDefaultFrameworkModel().getBeanFactory()
>             .getBean(FrameworkExecutorRepository.class).getSharedExecutor();
> ```

The ``FrameworkExecutorRepository`` class is available after ``3.0`` branch,  but my code is ``master`` branch

Reply:
get. Would you please submit a pr base on `3.0` branch after this pr merged?

Reply:
OK